src/patternfly/components/List/list.scss

```diff
@@ -22,6 +40,26 @@
     list-style: var(--pf-c-list--ul--ListStyle);
   }
 
+  .pf-c-list__item {
+    display: flex;
+  }
+
+  .pf-c-list__item-icon {
+    display: flex;
```

Comment:
Looks like this needs `flex-shrink: 0;`, otherwise it will shrink if there is a lot of content in the `item-text` element.

Reply:
Also needs `align-items: start;` or it will stretch whatever is in it vertically. You can see this if you put a non-square image in the `__item-icon` element, or put a square icon in and give it a modified `width`

Reply:
Actually, looking at this some more re: `align-items`, I'm not sure why this element has `display: flex;` set. If it's because the icons are too tall without `display: flex;`, it's because `__item-icon` will have a default line-height of `1.5`, which with a `16px` font, will make the height of the element `24px`. Setting `display: flex;` will give the box block formatting, removing the extra space created by `line-height`. A way to remove that extra height would be to set `line-height: 1;`, so the `line-height` will always match the current `font-size`. Try that and see if you can remove `display: flex;`, and then you won't need `align-items`, either.

Reply:
@mcoker It works, thank you!